apis/apis.json

```diff
@@ -226,12 +219,12 @@
     "dependencies": {
       "Google.Cloud.Diagnostics.Common": "project",
       "Google.Cloud.Diagnostics.AspNetCore.Analyzers": "1.0.0-beta02",
-      "Grpc.Core": "1.19.0"
+      "Grpc.Core": "1.20.1"
```

Comment:
Should references to `Grpc.Core` be `default`, rather than a specific version number?
`Google.Cloud.Firestore` uses `default`.

Reply:
Good point - this could absolutely be the default. It's fine to be default for any non-beta version. Will fix.

Reply:
I've added another commit for this; it doesn't change the project files (which is exactly what I expected). Will merge on green.